conf/modules.config

```
@@ -1665,19 +1580,20 @@ process {
     }
 
     withName: GATK_UNIFIEDGENOTYPER {
-        tag = { "${meta.reference}|${meta.sample_id}" }
-        ext.args = {[
-            "--sample_ploidy ${meta2.ploidy}",
-            "-stand_call_conf ${params.genotyping_gatk_call_conf}",
-            "-dcov ${params.genotyping_gatk_ug_downsample}",
-            "--output_mode ${params.genotyping_gatk_ug_out_mode}",
-            "--genotype_likelihoods_model ${params.genotyping_gatk_ug_genotype_mode}",
-            params.genotyping_gatk_ug_defaultbasequalities > 0 ? "--defaultBaseQualities ${params.genotyping_gatk_ug_defaultbasequalities}" : "", // Empty string since GATK complains if its default of -1 is provided.
-        ].join(' ').trim() }
+        tag        = { "${meta.reference}|${meta.sample_id}" }
+        ext.args   = {
+            [
+                "--sample_ploidy ${meta2.ploidy}",
```

this should be changed to behave the same as for FREEBAYES, since if the user is utilizing a multiref input, the ploidy can be left as an empty field in the meta.  (credit @jfy133 )

```suggestion
                "--sample-ploidy ${meta2.ploidy}",
                "            ref_meta.ploidy ? "--sample-ploidy ${ref_meta.ploidy}" : "--sample-ploidy 2", 

```